test/Mono.Linker.Tests.Cases/DataFlow/ArrayDataFlow.cs

```diff
@@ -148,6 +168,19 @@ static void TestGetElementAtUnknownIndex (int i = 0)
 			arr[i].RequiresPublicFields ();
 		}
 
+		// Trimmer code doesnt handle locals from different branches separetely, therefore merges incorrectly GetMethods with Unknown producing both warnings
```

Comment:
Would you mind filing an issue about this?

Reply:
Opened #2746